nais.yaml

```diff
@@ -2,20 +2,20 @@ apiVersion: "nais.io/v1alpha1"
 kind: "Application"
 metadata:
   name: sosialhjelp-innsyn
-  namespace: "q1"
+  namespace: {{namespace}}
   labels:
     team: digisos
 spec:
   image: "navikt/sosialhjelp-innsyn:{{version}}"
   port: 8080
   liveness:
-    path: /sosialhjelp/innsyn/internal/isAlive
+    path: /sosialhjelp/innsyn/
```

Comment:
Finnes det ikke eget isAlive-punkt for frontenden? 
(Prometheus etc, brukes ikke?)

Reply:
Jo, det finnes. :+1: